app/forms/declaration/DeclarationHolder.scala

```diff
@@ -61,6 +63,9 @@ object DeclarationHolder extends DeclarationPage {
       case _ => DeclarationHolder(Some(dividedString(0).trim), Some(Eori(dividedString(1).trim)))
     }
   }
+
+  override def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] =
+    Seq(TariffContentKey(s"tariff.declaration.addAuthorisationRequired.${defineJourneySpecificKey(decType)}"))
 }
 
 object DeclarationHolderRequired extends DeclarationPage
```

Comment:
There you have the page for which you created another one.

Reply:
Removed.